.gitignore

```diff
@@ -72,3 +72,6 @@ build/tests
 
 # E2E Test results
 testruns/
+
+# Virtualenv
+opencbdc_env/
```

Comment:
(optional) I think our root is already far too populous, and I see no benefit to this directory being visible by-default, nor does it seem obviously clarifying to name it `opencbdc_something`. I'd suggest renaming this directory to `.py_venv` or something similar.

Reply:
ACK, changed to that suggestion